prac_05/state_names.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+"""
+
+# Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+#  Write a loop that prints all of the states and names neatly lined up with string formatting
+for state, name in CODE_TO_NAME.items():
+    print('{:4} is {}'.format(state, name))
+
+
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    if state_code in CODE_TO_NAME:
+        print(state_code, "is", CODE_TO_NAME[state_code])
+    else:
+        print("Invalid short state")
+    state_code = input("Enter short state: ").upper()
+
+
```

Comment:
Lots of blank lines

Reply:
thanks for letting me know, will fix it now :)